libs/application/ui-fields/src/lib/ExpandableDescriptionFormField/ExpandableDescriptionFormField.tsx

```diff
@@ -18,7 +18,10 @@ export const ExpandableDescriptionFormField: FC<
   const { formatMessage, lang: locale } = useLocale()
 
   return (
-    <Box marginY={2}>
+    <Box
+      marginTop={field.marginTop ?? 2}
```

Comment:
Again maybe consider using destructuring with defaults rather than using the nullish coalescing operator here.
This is of course 100% ignorable, it just looks cleaner to me.

Reply:
Fair, done